src/components/VtTransition.vue

```diff
@@ -36,14 +36,15 @@ export default defineComponent({
         const { settings } = useSettings();
         const instance = getCurrentInstance();
 
-        const leave = (el: HTMLDivElement) => {
+        const leave = (el: Element) => {
```

Comment:
can you please do this for the other hooks too?

Reply:
Interestingly, in my editor (vscode), only `leave` had a type-error. It looks like the hyphenated events aren't properly working with the typescript language server.